packages/platform-android/src/commands/runAndroid/getApp.ts

```diff
@@ -0,0 +1,83 @@
+import xmlParser from 'fast-xml-parser';
```

Comment:
Um, we already use `xmldoc` to parse the manifest, check `readManifest.ts` file. Can you reuse it? I'd like to avoid parsing the file twice

Reply:
could I use [`fast-xml-parser`](https://github.com/NaturalIntelligence/fast-xml-parser#readme) to rewrite `readManifest.ts`?
reasons:
 - `fast-xml-parser` has better performance.  I try to parse a manifest by`xmldoc` in `RunKit`, page stuck and high CPU used, but `fast-xml-parser` not.
- result parsed by `fast-xml-parser` is much readable and programmable. 
- `fast-xml-parser` is long term maintained. `xmldoc` last update time is last year, but `fast-xml-parser` is last week
- `readManifest` is only used to get package name, refactor it will not have much impact on current code

Reply:
If it doesn't break stuff, then I'm good 👍

Reply:
@thymikee  refactor `readManifest.ts` and update relate test case